pypeit/history.py

```diff
@@ -163,6 +163,29 @@ def add_coadd1d(self, spec1d_files, objids, gpm_exp=None):
                         obj_info += f" {this_ext_header['MASKDEF_OBJNAME']}"
                 self.append(obj_info, add_date=False)
 
+    def add_coadd2d(self, spec2d_files:list[str], objname:str):
+        """
+        Add history entries for 2D coadding.
+        
+        The history shows what files and objects were used for coadding.
+        For example::
+            
+            HISTORY 2025-10-30T23:21 PypeIt Coadding target JWST in 4 spec2d files
+            HISTORY Exp 0: "spec2d_20251008.0053-JWST_DeVeny_20251008T045554.290.fits"
+            HISTORY Exp 1: "spec2d_20251008.0054-JWST_DeVeny_20251008T050103.240.fits"
+            HISTORY Exp 2: "spec2d_20251008.0055-JWST_DeVeny_20251008T050611.510.fits"
+            HISTORY Exp 3: "spec2d_20251008.0056-JWST_DeVeny_20251008T051119.790.fits"
+
+        Args:
+            spec2d_files (:obj:`list`): List of the spec2d files used for coadding.
+            objname (:obj:`str`): Name of the object being coadded
+        """
+        # Add history
+        self.append(f'PypeIt Coadded target {objname} in '
+                    f'{len(spec2d_files)} spec2d files')
+        for i, spec2d in enumerate(spec2d_files):
+            self.append(f'Exp {i}: "{Path(spec2d).name}"', add_date=False)
```

Comment:
Nit-picking, but (correct me if I'm wrong) spec2d files can be made of multiple combined exposures.  So using "Exp" can be misleading.  Can we use "File" instead?

Reply:
Yes, that's correct.  I was following the nomenclature in the logging... let me see about that...

Reply:
@kbwestfall : Both the history and the logging have been changed from "exp" to "file".